config/job_conf.xml.sample_advanced

```diff
@@ -205,6 +205,17 @@
                  zero (no execution) and the stderr/stdout of the k8s job is reported in galaxy (and the galaxy job set
                  to failed) -->
         </plugin>
+        <plugin id="godocker" type="runner" load="galaxy.jobs.runners.godocker:GodockerJobRunner">
```

Comment:
Can you put a line in here about what Godocker is and maybe a link to the platform.